examples/lines_bars_and_markers/barchart.py

```diff
@@ -7,32 +7,37 @@
 bars with labels.
 """
 
+# data from https://allisonhorst.github.io/palmerpenguins/
+
 import matplotlib.pyplot as plt
 import numpy as np
 
+species = ("Adelie", "Chinstrap", "Gentoo")
+measurements = ['Bill Length', 'Bill Depth']
+bill_length = (38.80, 48.83, 47.50)
+bill_depth = (18.34, 18.43, 14.98)
+penguin_means = [bill_length, bill_depth]
```

Comment:
I wanted to add the flipper length initially but because the sizes differ by a lot the bars become quite disproportionate in size (i.e bill length and depth become very small compared to the flipper length).  Here is an example:

![Figure_1](https://user-images.githubusercontent.com/73378227/203859879-4f16f015-6839-4b00-97f3-4a487ccb0cc9.png)

Can add if you'd like.

Reply:
What do you think?

![Figure_1](https://user-images.githubusercontent.com/73378227/203945352-39e47c38-dce6-4b39-9c69-fec3608c4af6.png)

Reply:
@story645

Reply:
No worries will push these changes to this one.